Simulation/6DOF_RK4/Executables/simulations/monte_python.py

```diff
@@ -42,40 +44,38 @@ def monte(x0, sim, rocket, atm, dt, kf_dt, samples, output_folder, target_size,
     t_start = time.time()
     # Nominal run
     print("Calculating nominal trajectory")
-    record = pysim.pysim(dt, kf_dt)
+    record = pysim.pysim(np.zeros((6,3)), dt, kf_dt)
     data_process.save_record(record, os.path.join(output_folder, "SimData", "nominal.csv"), target_size=target_size)
 
     # Monte Carlo runs
     for i in range(samples):
         print(f"Running sample {i+1} of {samples}")
+        atm, rocket = simulation.gen_sim_objs(dataloader.config, kwargs)
+        sim = propagtor.Simulator(atm=atm, rocket=rocket)
+        x0 = np.zeros((6,3))
+        x0[3:6] = np.random.normal(kwargs.get("launch_angle_mean", 0), kwargs.get("launch_angle_stddev",0), (3,))
         pysim.simulator(x0, sim, rocket, atm, dt, kf_dt)
         record = rocket.to_list()
         data_process.save_record(record, os.path.join(output_folder, "SimData", f"sim_data_{i}.csv"), target_size=target_size)
     t_end = time.time() - t_start
     print(f"Monte Python Runtime: {t_end:.2f} seconds")
 
-
 config = dataloader.config
-x0 = np.zeros((6,3))
-x0[3] = np.array([0.0, 0.05, 0.01])
 dt = 0.1
 kf_dt = 0.1
 
 target_size = 880
-samples = 2
-
-atm = atmosphere.Atmosphere(wind_direction_variance_stddev=0.2, 
-                            wind_magnitude_variance_stddev=3.0,
-                            enable_direction_variance=True,
-                            enable_magnitude_variance=True, 
-                            nominal_wind_magnitude=2,
-                            nominal_wind_direction=np.array([0,1,0]))
-stages = []
-for stage in config['rocket']['stages'][1:]:
-    stages.append(rocket_model.Rocket(stage, atm=atm))
-rocket = rocket_model.Rocket(config['rocket']['stages'][0], atm=atm, stages=stages)
+samples = 10
 
-sim = propagtor.Simulator(atm=atm, rocket=rocket)
 output_folder = os.path.join(os.path.dirname(__file__), config["meta"]["monte_carlo_output_folder"])
 
-monte(x0, sim, rocket, atm, dt, kf_dt, samples, output_folder, target_size)
\ No newline at end of file
+sim_params = {"wind_direction_stddev": 0,
+               "wind_magnitude_stddev": 10,
+               "enable_direction_variance": True,
+               "enable_magnitude_variance": True,
+               "nominal_wind_magnitude": 0.0,
+               "nominal_wind_direction": np.array([0, 1.0, 0]),
+               "launch_angle_mean": .00,
+               "launch_angle_stddev": 0.00}
+
+monte(dt, kf_dt, samples, output_folder, target_size, **sim_params)
```

Comment:
idk if out of scope but idt this code is thread safe so if we wanna ramp up number of runs in parallel we need to do that